packages/grafana-e2e/src/flows/deleteDashboard.ts

```diff
@@ -1,12 +1,20 @@
 import { e2e } from '../index';
 import { fromBaseUrl } from '../support/url';
 
-export const deleteDashboard = (dashBoardUid: string) => {
-  e2e().logToConsole('Deleting dashboard with uid:', dashBoardUid);
-  e2e().request('DELETE', fromBaseUrl(`/api/dashboards/uid/${dashBoardUid}`));
+export interface DeleteDashboardConfig {
+  title: string;
+  uid: string;
+}
+
+export const deleteDashboard = ({ title, uid }: DeleteDashboardConfig) => {
+  e2e().logToConsole('Deleting dashboard with uid:', uid);
+
+  // Avoid dashboard page errors
+  e2e.pages.Home.visit();
+  e2e().request('DELETE', fromBaseUrl(`/api/dashboards/uid/${uid}`));
 
   /* https://github.com/cypress-io/cypress/issues/2831
```

Comment:
I think it's safe to remove this commented out section now.

Reply:
Will we not be reverting to it at some point in the future, though? It seems better to use the UI than a request.